src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -241,6 +241,8 @@ internal bool IsOutOfProc
             }
         }
 
+        public bool ParallelTasksSucceeded { get; private set; } = true;
```

Comment:
This is a bad name, but I had some trouble with it. Please opine.

Reply:
`BuildRequestFailed`?